flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/connectors/DynamicSinkUtils.java

```diff
@@ -209,6 +213,78 @@ public static RelNode convertSinkToRel(
                 sink);
     }
 
+    /**
+     * Converts a given {@link DynamicTableSink} to a {@link RelNode}. It adds helper projections if
+     * necessary.
+     */
+    public static RelNode convertCtasToRel(
```

Comment:
```suggestion
    public static RelNode convertCreateTableAsToRel(
```
avoid abbreviations, we cannot assume that everybody knows them

Reply:
Done